starlette/testclient.py

```diff
@@ -288,7 +288,7 @@ async def receive() -> Message:
                     await response_complete.wait()
                 return {"type": "http.disconnect"}
 
-            body = request.read()
+            body = await anyio.to_thread.run_sync(request.read)
```

Comment:
Httpx `request.read` can be blocking (eg: reading multipart file).
`await request.aread()` can't be used as-is since the request `stream` is a `SyncByteStream` not an `AsyncByteStream`.

Reply:
IMO, this is not a problem in unit tests and we can neglect it.